service.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import json
+import time
+
+import falcon
+import jsonpickle as jsonpickle
+import onnx
+import onnxruntime as ort
+from torch import nn
+
+PORT_NUMBER = 8080
+start = time.time()
+
+# Load the ONNX model
+model = onnx.load("model.onnx")
+
+# Check that the IR is well formed
+onnx.checker.check_model(model)
+
+so = ort.SessionOptions()
+so.graph_optimization_level = ort.GraphOptimizationLevel.ORT_ENABLE_ALL
```

Comment:
what does this do?

Reply:
There is a good explanation in the ONNX docs [here](https://github.com/microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Graph_Optimizations.md)